### ISSUE TITLE
GUI polish - click in search box

### ISSUE THREAD

**Issue by synctext** [Tribler/tribler#6260]
No feedback to the user is provided when the keyboard focus is inside the search box. The 'tutorial' background text should be cleared. We force the start of typing in a box that already contains text, this induces anxiety.

**Comment by devos50** [comment]
> We force the start of typing in a box that already contains text, this induces anxiety.

This is placeholder text and is a pretty common design pattern implemented by many interfaces used at scale (YouTube, Spotify etc).

> No feedback to the user is provided when the keyboard focus is inside the search box.

Valid feedback. Tribler removes all focus rectangles when starting since every element by default gets one (and that clutters the user interface).

We could highlight the search box when it has focus with a trivial stylesheet:

<img width="556" alt="Schermafbeelding 2021-10-05 om 12 07 26" src="https://user-images.githubusercontent.com/1707075/136005228-2e4f9cdf-f9e2-4cea-867d-44f52910f544.png">

@synctext If we agree on this, I can file a PR 👍

**Comment by synctext** [comment]
Great polish idea, yeah
